<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1853</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tab_Widget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1841</width>
     <height>781</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="View_Table">
    <attribute name="title">
     <string>View Table</string>
    </attribute>
    <widget class="QPushButton" name="Home_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>680</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Read_In_From_File_Button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>590</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Read In from File</string>
     </property>
    </widget>
    <widget class="QTableView" name="Admin_Datatable">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>1371</width>
       <height>721</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Surface Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Opened</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Roof Type</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Add_Seating_Capacity_SpinBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>200000</number>
     </property>
    </widget>
    <widget class="QComboBox" name="Add_Conference_ComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Team Name</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Add_Division_ComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Division</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Seating Capacity</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Add_Team_Name_LineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Add_Roof_Type_ComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>360</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadium Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Add_Location_LineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Add_Stadium_Name_LineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Add_Surface_Type_ComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Conference</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Add_Year_SpinBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>410</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1924</number>
     </property>
     <property name="maximum">
      <number>2100</number>
     </property>
     <property name="value">
      <number>2020</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Add_Team_PushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Team</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Edit_Souvenirs">
    <attribute name="title">
     <string>Edit Souvenirs</string>
    </attribute>
    <widget class="QTableView" name="Update_Souvenir_Datatable">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>1331</width>
       <height>711</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Souvenir_Name_LineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Price</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Price_Double_SpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <double>2000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QComboBox" name="Team_Name_ComboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Team Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Update_Souvenir_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Souvenir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Add_Souvenir_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Souvenir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Delete_Souvenir_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete Souvenir</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Edit_Stadium_Info">
    <attribute name="title">
     <string>Edit Stadium Info</string>
    </attribute>
    <widget class="QTableView" name="Edit_Stadium_TableView">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>1321</width>
       <height>711</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Team Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadium Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Seating Capacity</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Conference</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Division</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Surface Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Roof Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Opened</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Team_Name_LineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Stadium_Name_LineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="Seating_Capacity_SpinBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>200000</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="Location_LineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Conference_Combo_Box">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Division_Combo_Box">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Surface_Type_ComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Roof_Type_ComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>380</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="Year_SpinBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>430</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1924</number>
     </property>
     <property name="maximum">
      <number>2100</number>
     </property>
     <property name="value">
      <number>2020</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Update_Stadium_PushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Stadium Info</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
